### DISCUSSION TITLE
[Art competition] Classic Cryptography

### DISCUSSION THREAD

**Discussion by uziahmd** [prosyslab-classroom/cs348-information-security#401] [Art competition]
![DALL·E 2023-03-24 12 44 41 - A super realistic looking animated futuristic robot with tensed features that is passionately painting the wooden blocks infront of him with different](https://github.com/prosyslab-classroom/cs348-information-security/assets/44532534/c6613c92-6db0-4fb6-af0f-6a8212b00dce)

## Short Description
For me, [*the chosen concept*] means [*A super realistic looking animated futuristic robot with tensed features that is passionately painting the wooden blocks with different colors while hiding their original colors after getting instruction from the chart in front of him before sending the blocks out of the room on a conveyor belt.*]".

## Long Description (up to 3 sentences)

Once in a workshop, a lifelike robot diligently painted wooden blocks with different colors, while hiding their original hues. Guided by a chart, the robot concealed the true colors beneath captivating layers of paint. This mirrors classical cryptography, where secret methods are used to encode messages, safeguarding their meaning from prying eyes.

**Comment by KihongHeo** [comment]
Good. It is somewhat unfortunate that DALL-E does not clearly understand your intention.
